### DISCUSSION TITLE
Lspsaga color problem

### DISCUSSION THREAD

**Discussion by pogopaule** [EdenEast/nightfox.nvim#300] [Q&A] [answered]
I struggle to debug the following problem:

I ported [my config](https://github.com/pogopaule/dotfiles/tree/nix/nvim/lua/config) to [lazy](https://github.com/folke/lazy.nvim). Since then all popups from Lspsaga have the wrong color.
![image](https://user-images.githubusercontent.com/576949/212933589-a4c99b43-f8d8-4964-b05f-dcc992cfe933.png)

When I enter `:colorscheme dawnfox` the color issue is fixed:
![image](https://user-images.githubusercontent.com/576949/212934118-cce5a77a-473a-4d3c-8ef9-35a214d67635.png)

I tried various different combinations of when Lspsaga and nightfox load. Nothing fixed the issue.
The theme tokyonight does not have this issue.

These are the relevant files
[colorscheme.lua](https://github.com/pogopaule/dotfiles/blob/7f5f7427e9efa637221c7d653593473c9dca229e/nvim/lua/config/plugins/colorscheme.lua)
[lsp.lua](https://github.com/pogopaule/dotfiles/blob/7f5f7427e9efa637221c7d653593473c9dca229e/nvim/lua/config/plugins/lsp.lua#L105-L126)

**Comment by EdenEast** [comment, marked as answer]
This looks like lspsaga has changed how they are handling highlight groups in pr [586](https://github.com/glepnir/lspsaga.nvim/pull/586).

**Reply by pogopaule** [reply]
Ok I'll try to fix it and open a PR. When I start working on it I'll let you know here in this thread.
